test/simulator/timestamp/simulator_interface/simulator_interface.cpp

```diff
@@ -532,6 +532,7 @@ print_rules()
         switch (choice) {
         case 1:
             print_border_msg("Timestamp value should be greater than 0.", WHITE);
+            /* TODO */
```

Comment:
@Siddhartha8899, what should we do here? I am not sure where that message is used.

Reply:
As discussed with Sid, we simply need to update the rules here and say it is no longer ok to do this.